docs/core/run-time-config/compilation.md

```diff
@@ -125,3 +125,13 @@ Project file:
 
 </Project>
 ```
+
+## ReadyToRun
+
+- Configures whether the .NET Core runtime uses pre-compiled images for framework code (and user assemblies, if pre-compiled images are available). Disabling this option forces the runtime to JIT-compile framework code.
```

Comment:
Is the part about user assemblies true, or are the R2R images always used if they're available?

Reply:
There is no distinction between user/framework assemblies for this variable